content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -402,6 +402,14 @@ In the past, the annotation `volume.beta.kubernetes.io/mount-options` was used i
 of the `mountOptions` attribute. This annotation is still working, however
 it will become fully deprecated in a future Kubernetes release.
 
+### Node Affinity
+
+{{< note >}}
+In most cases, you do not need to set this field. It is automatically populated for [AWS EBS](/docs/concepts/storage/volumes/#awselasticblockstore), [GCE PD](/docs/concepts/storage/volumes/#gcepersistentdisk) and [Azure Disk](/docs/concepts/storage/volumes/#azuredisk) volume block types. You need to explicitly set this in case of [local](/docs/concepts/storage/volumes/#local) volumes. Most other volume types do not need this field.
```

Comment:
"In most cases" => "For most volume types"

"in case of local" => "for local"

Reply:
addressed the comments